dev/com.ibm.ws.request.probe.audit.servlet/src/com/ibm/ws/request/probe/audit/servlet/AuditPE.java

```diff
@@ -239,6 +240,9 @@ public void processCounter(Event event) {
 				case APPLICATION_PASSWORD_TOKEN_01:
 					auditEventApplicationPasswordToken(methodParams);
 					break;
+				case SECURITY_SAF_AUTHZ_DETAILS:
```

Comment:
I think that this is supposed to be the non-details one, because the details one already exists just above here?

Reply:
Fixed, i think that got messed up when I merged with Emily's changes.